### DISCUSSION TITLE
A color damage setting like in Head Switching

### DISCUSSION THREAD

**Discussion by Glebka73a1** [ntsc-rs/ntsc-rs#199] [Ideas]
Head Switching on the bottom has color damage based on Horizontal Shift. It's best visible at the maximum height. I'd like to see a setting that would allow that color damage, maybe with the same horizontal shift but for the entire screen. This would help make very realistic VHS tape glitches.
![image](https://github.com/user-attachments/assets/7a6ef0ef-973f-4df1-a748-c227a08c889b)

**Comment by notdigidani** [comment]
lets hope @Degamisu see this and adds this 🙏🙏
